assets/_locales/de/messages.json

```diff
@@ -9,6 +9,7 @@
 "cj_i18n_06858":{"message":"Alle Bildraten"},
 "cj_i18n_06860":{"message":"Unterstütze den Entwickler"},
 "cj_i18n_06861":{"message":"Bewerte die Erweiterung"},
-"cj_i18n_06862":{"message":"Nutze die selbe Qualität für alle Bildraten"},
-"cj_i18n_07266":{"message":"Erweiterung ist eingeschaltet"}
```

Comment:
I was informed that "Erweiterung ist eingeschaltet" is the correct translation
Everything else was merged into the [spreadsheet](https://github.com/avi12/youtube-auto-hd#translating)
I'd appreciate it if next time that you change a message, it'd be done directly in this spreadsheet as it's used as a part of the workflow